File: examples/nodejs/route.js
import { Techkern } from "@techkern/sdk";
const k = new Techkern({ apiKey: process.env.TECHKERN_KEY });
const r = await k.route({ model: "auto", messages: [{ role: "user", content: "Hi" }] });
console.log(r.text, r.routedTo);

File: examples/python/route.py
import asyncio, os
from techkern_sdk import Techkern

async def main():
    k = Techkern(api_key=os.environ["TECHKERN_KEY"])
    r = await k.route(model="auto", messages=[{"role":"user","content":"Hi"}])
    print(r)

asyncio.run(main())
